app/models/tenant.rb

```diff
@@ -155,7 +155,7 @@ def set_quotas(quotas)
       # Delete any quotas that were not passed in
       tenant_quotas.destroy_missing(updated_keys)
       # unfortunatly, an extra scope is created in destroy_missing, so we need to reload the records
-      clear_association_cache
+      self.instance_variable_set :@association_cache, {}
```

Comment:
Added in: 37862630cab901e2b6dc4871aaa17d39619a6ac9

Again, not sure if we need to do this.

Reply:
I think this one can also just be `self.reload` or just `reload`